src/components/Contracts/index.tsx

```diff
@@ -0,0 +1,236 @@
+        divisible or not.
+      </Translate>
+    ),
+  },
+  {
+    name: '🎨 LSP8 Identifiable Digital Asset',
+    url: 'https://docs.lukso.tech/contracts/overview/DigitalAssets/#lsp8-identifiable-digital-asset',
+    urlABI:
+      'https://docs.lukso.tech/contracts/contracts/LSP8IdentifiableDigitalAsset/LSP8IdentifiableDigitalAsset.md',
+    description: (
+      <Translate id="playground.codesandbox.description">
+        Creates a non-fungible token where each NFT can be represented
+        differently (numbers, serial numbers, an ERC725Y contract per NFT...).
+      </Translate>
+    ),
+  },
+];
+
+const LSPs_OwnershipAccessControl = [
+  {
+    name: '🔐 LSP6 Key Manager',
+    url: 'https://docs.lukso.tech/contracts/overview/KeyManager',
+    urlABI: 'https://docs.lukso.tech/contracts/contracts/LSP6KeyManager',
+    description: (
+      <Translate id="playground.codesandbox.description">
+        Allows multi-control over the account using different permissions.
+      </Translate>
+    ),
+  },
+  {
+    name: '🌱 LSP11 Basic Social Recovery',
+    url: '',
+    urlABI:
+      'https://docs.lukso.tech/contracts/contracts/LSP11BasicSocialRecovery/',
+    description: (
+      <Translate>
+        Recover access to your Blockchain based account if you are locked using
+        a social recovery mechanism.
+      </Translate>
+    ),
+  },
+  {
+    name: '2️⃣ LSP14 Ownable 2 Steps',
+    url: '',
+    urlABI: 'https://docs.lukso.tech/contracts/contracts/LSP14Ownable2Step/',
+    description: (
+      <Translate>
+        A module to allow secure owner management of a smart contract using a
+        2-steps process for transferring and renouncing ownership.
+      </Translate>
+    ),
+  },
+];
+
+interface Props {
+  name: string;
+  url: string;
+  urlABI: string;
+  description: JSX.Element;
+}
+
+function ContractCard({ name, url, urlABI, description }: Props) {
+  const displayLink: boolean = url === '';
+
+  return (
+    <div className={clsx('card')}>
+      <div className="card__body">
+        <Heading as="h3">{name}</Heading>
+        <p>{description}</p>
+      </div>
+      <div className="card__footer">
+        <div className="button-group button-group--block">
+          <a
+            style={{ visibility: url == '' ? 'hidden' : 'visible' }}
+            className="button button--secondary"
+            href={url}
+          >
+            Overview
+          </a>
+          <a
+            style={{ visibility: urlABI == '' ? 'hidden' : 'visible' }}
+            className="button button--secondary"
+            href={urlABI}
+          >
+            ABI Reference
+          </a>
+        </div>
+      </div>
+    </div>
+  );
+}
+
+export function ContractCardsRow(): JSX.Element {
+  return (
+    <>
+      <div className="row">
+        <div className="col col--6 margin-bottom--lg">
+          <h2>Ownership & Access Control</h2>
+          {LSPs_OwnershipAccessControl.map((playground) => (
```

Comment:
what is playground?

What does it mean?

Usually when doing a map, i name things like this

```
users.map((user)
```

Like one thing using plural and the map param in singular. But they have the same name.